libraries/AP_GPS/AP_GPS_FEMTO.cpp

```diff
@@ -0,0 +1,310 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//  Femtomes GPS driver for ArduPilot.
+//  Code by Rui Zheng <ruizheng@femtomes.com>
+
+
+#include "AP_GPS.h"
+#include "AP_GPS_FEMTO.h"
+#include <AP_Logger/AP_Logger.h>
```

Comment:
this import is unused

Reply:
ok，I will move it to FEMTO_DEBUGGING place